<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="QANR90JHUTD" name="LoginTest-ff" tests="2" failures="1" timestamp="1 Jun 2018 08:58:24 GMT" time="120.374" errors="0">
  <testcase name="negativeLoginTest" time="40.438" classname="com.qbr.TestSteps.LoginTest">
    <failure type="java.lang.AssertionError" message="Login Err Msg different. 
Expected:Oops! The login details you entered do not match our records.
Actual:Oops! The login details you entered do not match our records. Please check your details and re-enter.
Please note your user account will be locked out after three login attempts.
Please click the Forgot Password? link to change your password and unlock your user account.
If you need assistance please contact the Qantas Business Rewards Service Centre on 13 74 78. expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: Login Err Msg different. 
Expected:Oops! The login details you entered do not match our records.
Actual:Oops! The login details you entered do not match our records. Please check your details and re-enter.
Please note your user account will be locked out after three login attempts.
Please click the Forgot Password? link to change your password and unlock your user account.
If you need assistance please contact the Qantas Business Rewards Service Centre on 13 74 78. expected [true] but found [false]
	at com.qbr.TestSteps.LoginTest.negativeLoginTest(LoginTest.java:57)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:748)
... Removed 19 stack frames]]>
    </failure>
  </testcase> <!-- negativeLoginTest -->
  <testcase name="positiveLoginTest" time="45.714" classname="com.qbr.TestSteps.LoginTest"/>
</testsuite> <!-- LoginTest-ff -->
